### PR TITLE
electron: 1.7.11 -> 1.8.2

### PR DESCRIPTION
**Pull request by manveru**
###### Motivation for this change

Needed for https://github.com/NixOS/nixpkgs/pull/35334

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by volth** [comment]
```1.8.1``` broke Rambox
https://github.com/NixOS/nixpkgs/pull/32398
Is it fixed in ```1.8.2```?

**Comment by volth** [comment]
Also, please add ```aarch64```.
it is supported since ```1.8.x```

**Comment by manveru** [comment]
I can confirm that Rambox at least starts up fine. Also added the arm64 version.

File: pkgs/development/tools/electron/default.nix [hunk 1]
```diff
@@ -20,15 +20,19 @@ let
...
         url = "https://github.com/electron/electron/releases/download/v${version}/electron-v${version}-linux-x64.zip";
-        sha256 = "0v22xhzbq9lcbc83laqs45pbx8gzv648qfkj01pxfsmv3lb4myrl";
+        sha256 = "07ggq9wgfz3z5z0lwzzgs6im0qs83pz0pcfwr0r42zgmwg7j78b8";
       };
       armv7l-linux = fetchurl {
         url = "https://github.com/electron/electron/releases/download/v${version}/electron-v${version}-linux-armv7l.zip";
-        sha256 = "02n2y69zwzacigqp6f4vg47cmjzf8gvbbispmzkg3pnzk4qc9473";
+        sha256 = "1b0p5x9zigyd6d8gz2hxc4scllrpnbx1dzzwlsvw6ilqbj1ypc7i";
+      };
+      arm64-linux = fetchurl {
```
Comment (volth) [inline pkgs/development/tools/electron/default.nix]: ```arm64-linux``` -> ```aarch64-linux``` or it will crash

**Comment by manveru** [comment]
amended.

File: pkgs/development/tools/electron/default.nix (hunk 1 above)
Comment (volth) [reply pkgs/development/tools/electron/default.nix]: please add "aarch64-linux" to ```meta.platforms``` at line 14 (github does allow to comment there)

**Comment by volth** [comment]
thank you!
The last thing - please add "aarch64-linux" to ```meta.platforms``` at line 14 (github does allow to comment there because it is too far from the changed lines)

**Comment by manveru** [comment]
Alright, did that as well as squash everything into a single commit again. Hope it builds now :)

**Pull request merged**